### PR TITLE
Jupyter Notebooks with Amazon Transcribe (+Medical) and Claude 3 model family

### PR DESCRIPTION
**Pull request by gattil**
*Description of changes:*

1. Two new Jupyter Notebooks that use Amazon Transcribe (+Medical) and Claude 3 Sonnet/Haiku models to generate clinical plans from audio recordings.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: ClinicalPlansWithClaude3_AmazonTranscribe.ipynb [hunk 1]
```diff
@@ -0,0 +1,1830 @@
...
+    "ExecuteTime": {
+     "end_time": "2024-03-20T12:57:12.949994Z",
+     "start_time": "2024-03-20T12:57:12.941044Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "timestamp = datetime.datetime.now().strftime(\"%Y-%m-%d-%H%M%S\")\n",
+    "media_uri = \"s3://%s/%s/%s/%s\" % (bucket, prefix, \"Audio Recordings\", object_name)\n",
+    "job_name = \"trascribe-%s-%s\" % (object_name.split(\".\")[0],timestamp)"
```
Comment (scottschreckengaust) [inline ClinicalPlansWithClaude3_AmazonTranscribe.ipynb]:
```suggestion
    "job_name = \"transcribe-%s-%s\" % (object_name.split(\".\")[0],timestamp)"
```

File: ClinicalPlansWithClaude3_AmazonTranscribe.ipynb [hunk 2]
```diff
@@ -0,0 +1,1830 @@
...
+     "start_time": "2024-03-20T12:57:14.962568Z"
+    },
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{'TranscriptionJob': {'TranscriptionJobName': 'trascribe-RES0038-2024-03-25-112948', 'TranscriptionJobStatus': 'IN_PROGRESS', 'LanguageCode': 'en-US', 'Media': {'MediaFileUri': 's3://sagemaker-us-east-1-563836490331/rawdata/Audio Recordings/RES0038.mp3'}, 'StartTime': datetime.datetime(2024, 3, 25, 11, 29, 48, 489000, tzinfo=tzlocal()), 'CreationTime': datetime.datetime(2024, 3, 25, 11, 29, 48, 464000, tzinfo=tzlocal()), 'Settings': {'ShowSpeakerLabels': True, 'MaxSpeakerLabels': 2, 'ChannelIdentification': False}}, 'ResponseMetadata': {'RequestId': 'c71cbc64-84e4-4002-a0ac-a1e8d11a3e73', 'HTTPStatusCode': 200, 'HTTPHeaders': {'x-amzn-requestid': 'c71cbc64-84e4-4002-a0ac-a1e8d11a3e73', 'content-type': 'application/x-amz-json-1.1', 'content-length': '397', 'date': 'Mon, 25 Mar 2024 11:29:48 GMT'}, 'RetryAttempts': 0}}\n"
```
Comment (scottschreckengaust) [inline ClinicalPlansWithClaude3_AmazonTranscribe.ipynb]:
```suggestion
      "{'TranscriptionJob': {'TranscriptionJobName': 'transcribe-RES0038-2024-03-25-112948', 'TranscriptionJobStatus': 'IN_PROGRESS', 'LanguageCode': 'en-US', 'Media': {'MediaFileUri': 's3://sagemaker-us-east-1-563836490331/rawdata/Audio Recordings/RES0038.mp3'}, 'StartTime': datetime.datetime(2024, 3, 25, 11, 29, 48, 489000, tzinfo=tzlocal()), 'CreationTime': datetime.datetime(2024, 3, 25, 11, 29, 48, 464000, tzinfo=tzlocal()), 'Settings': {'ShowSpeakerLabels': True, 'MaxSpeakerLabels': 2, 'ChannelIdentification': False}}, 'ResponseMetadata': {'RequestId': 'c71cbc64-84e4-4002-a0ac-a1e8d11a3e73', 'HTTPStatusCode': 200, 'HTTPHeaders': {'x-amzn-requestid': 'c71cbc64-84e4-4002-a0ac-a1e8d11a3e73', 'content-type': 'application/x-amz-json-1.1', 'content-length': '397', 'date': 'Mon, 25 Mar 2024 11:29:48 GMT'}, 'RetryAttempts': 0}}\n"
```

File: ClinicalPlansWithClaude3_AmazonTranscribe.ipynb [hunk 3]
```diff
@@ -0,0 +1,1830 @@
...
+      "Not ready yet...\n",
+      "Not ready yet...\n",
+      "Not ready yet...\n",
+      "Not ready yet...\n",
+      "Not ready yet...\n",
+      "Not ready yet...\n",
+      "Not ready yet...\n",
+      "Not ready yet...\n",
+      "Not ready yet...\n",
+      "Job status: trascribe-RES0038-2024-03-25-112948\n",
```
Comment (scottschreckengaust) [inline ClinicalPlansWithClaude3_AmazonTranscribe.ipynb]:
```suggestion
      "Job status: transcribe-RES0038-2024-03-25-112948\n",
```

File: ClinicalPlansWithClaude3_AmazonTranscribe.ipynb [hunk 4]
```diff
@@ -0,0 +1,1830 @@
...
+      "Job duration: 0:00:51.405000\n",
+      "Transcription file: https://s3.us-east-1.amazonaws.com/sagemaker-us-east-1-563836490331/trascribe-RES0038-2024-03-25-112948.json\n"
```
Comment (scottschreckengaust) [inline ClinicalPlansWithClaude3_AmazonTranscribe.ipynb]:
```suggestion
      "Transcription file: https://s3.us-east-1.amazonaws.com/sagemaker-us-east-1-563836490331/transcribe-RES0038-2024-03-25-112948.json\n"
```

File: ClinicalPlansWithClaude3_AmazonTranscribeMedical.ipynb [hunk 5]
```diff
@@ -0,0 +1,1307 @@
...
+     "start_time": "2024-03-20T12:57:14.962568Z"
+    },
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{'MedicalTranscriptionJob': {'MedicalTranscriptionJobName': 'trascribe-RES0038-2024-03-25-113438', 'TranscriptionJobStatus': 'IN_PROGRESS', 'LanguageCode': 'en-US', 'Media': {'MediaFileUri': 's3://sagemaker-us-east-1-563836490331/rawdata/Audio Recordings/RES0038.mp3'}, 'StartTime': datetime.datetime(2024, 3, 25, 11, 34, 38, 526000, tzinfo=tzlocal()), 'CreationTime': datetime.datetime(2024, 3, 25, 11, 34, 38, 497000, tzinfo=tzlocal()), 'Settings': {'ShowSpeakerLabels': True, 'MaxSpeakerLabels': 2, 'ChannelIdentification': False}, 'Specialty': 'PRIMARYCARE', 'Type': 'CONVERSATION'}, 'ResponseMetadata': {'RequestId': 'f5ae1484-bfdf-45b4-8642-78e36c2ee0b2', 'HTTPStatusCode': 200, 'HTTPHeaders': {'x-amzn-requestid': 'f5ae1484-bfdf-45b4-8642-78e36c2ee0b2', 'content-type': 'application/x-amz-json-1.1', 'content-length': '459', 'date': 'Mon, 25 Mar 2024 11:34:38 GMT'}, 'RetryAttempts': 0}}\n"
```
Comment (scottschreckengaust) [inline ClinicalPlansWithClaude3_AmazonTranscribeMedical.ipynb]:
```suggestion
      "{'MedicalTranscriptionJob': {'MedicalTranscriptionJobName': 'transcribe-RES0038-2024-03-25-113438', 'TranscriptionJobStatus': 'IN_PROGRESS', 'LanguageCode': 'en-US', 'Media': {'MediaFileUri': 's3://sagemaker-us-east-1-563836490331/rawdata/Audio Recordings/RES0038.mp3'}, 'StartTime': datetime.datetime(2024, 3, 25, 11, 34, 38, 526000, tzinfo=tzlocal()), 'CreationTime': datetime.datetime(2024, 3, 25, 11, 34, 38, 497000, tzinfo=tzlocal()), 'Settings': {'ShowSpeakerLabels': True, 'MaxSpeakerLabels': 2, 'ChannelIdentification': False}, 'Specialty': 'PRIMARYCARE', 'Type': 'CONVERSATION'}, 'ResponseMetadata': {'RequestId': 'f5ae1484-bfdf-45b4-8642-78e36c2ee0b2', 'HTTPStatusCode': 200, 'HTTPHeaders': {'x-amzn-requestid': 'f5ae1484-bfdf-45b4-8642-78e36c2ee0b2', 'content-type': 'application/x-amz-json-1.1', 'content-length': '459', 'date': 'Mon, 25 Mar 2024 11:34:38 GMT'}, 'RetryAttempts': 0}}\n"
```

File: ClinicalPlansWithClaude3_AmazonTranscribeMedical.ipynb [hunk 6]
```diff
@@ -0,0 +1,1307 @@
...
+    "ExecuteTime": {
+     "end_time": "2024-03-20T12:57:12.949994Z",
+     "start_time": "2024-03-20T12:57:12.941044Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "timestamp = datetime.datetime.now().strftime(\"%Y-%m-%d-%H%M%S\")\n",
+    "media_uri = \"s3://%s/%s/%s/%s\" % (bucket, prefix, \"Audio Recordings\", object_name)\n",
+    "job_name = \"trascribe-%s-%s\" % (object_name.split(\".\")[0],timestamp)"
```
Comment (scottschreckengaust) [inline ClinicalPlansWithClaude3_AmazonTranscribeMedical.ipynb]:
```suggestion
    "job_name = \"transcribe-%s-%s\" % (object_name.split(\".\")[0],timestamp)"
```

**Review by scottschreckengaust — CHANGES_REQUESTED** [review]
Minor correction

**Comment by gattil** [comment]
@scottschreckengaust Good catch! Thanks for preparing all the suggestions. ;-)

**Review by scottschreckengaust — APPROVED** [review]

**Pull request merged**